pkg/monitoring/config/parse.go

```diff
@@ -21,6 +21,20 @@ func Parse() (Config, error) {
 	return cfg, err
 }
 
+func ParseWithoutKafka() (Config, error) {
```

Comment:
i'd refactor this so they share the same initialization helper and only apply different validation ~
~
```
func defaultCfg() Config, error {
cfg := Config{}

	if err := parseEnvVars(&cfg); err != nil {
		return cfg, err
	}

	applyDefaults(&cfg)
}

func Parse() Config, error {
 cfg, err := defaultCfg()
 if err != nil {
 	return Config{}, fmt.Errorf("failed to create default configuration: %w", err)
 	}
 	return cfg, validateConfig(cfg)
 	}
 	
 func ParseWithoutKafka error ...
 return cfg, validateConfigWithoutKafka
 ```